### PR TITLE
Font sizing, site width, dropdown nav items

### PR DESCRIPTION
**Pull request by daviddarnes**
- Adjust font sizing
- Adjust site width size
- Add dropdown nav to header nav

### CODE REVIEW

File: assets/styles.scss [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+@import "alembic";
+
+// Notification bar
+.notification-bar {
+  text-align: center;
+  padding: .75rem 1.25rem;
+  margin-bottom: 1rem;
+  background-color: $codeBackgroundColour;
```
Comment (odow) [inline assets/styles.scss]: Hmm, so I tried changing the color here to `"red"` and `#f00` without success. Is it possible to make the banner background orange (or dark-grey) to stand out? What am I missing?

File: _layouts/page-with-notification.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+---
+
+{%- capture notification %}
+Testing
```
Comment (mlubin) [inline _layouts/page-with-notification.html]: Could this be a value that's defined in the config file?

File: _config.yml [hunk 3]
```diff
@@ -72,12 +72,13 @@ collections:
...
-#      path: ""
-#    values:
-#      image: "/assets/default-social-image.png" # Default image for sharing
+defaults:
+  -
+    scope:
+      path: ""
+      types: "pages"
+    values:
+      layout: "page-with-notification"
```
Comment (mlubin) [inline _config.yml]: To follow up on @odow's question, could you add a short comment on how to switch the notification bar on and off?

File: _includes/nav-header.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  </ul>
+  <button class="button  button--nav" aria-label="Menu toggle">
+    {% include icon.html id="nav" title="Menu" %}
+  </button>
+</nav>
+{% else %}
+  {% include nav-default.html %}
+{% endif %}
+
+<script type="text/javascript">
```
Comment (mlubin) [inline _includes/nav-header.html]: Mostly out of curiosity, what is this javascript doing? A short comment would suffice.

File: _config.yml (hunk 3 above)
Comment (odow) [reply _config.yml]: I figured it out. You just comment out this section.

**Comment by mlubin** [comment]
There's an issue with drop-down menu expanding and unexpanding in response to a window resize:
jump dev
It seems to be a new issue, doesn't happen with the current jump.dev.

File: assets/styles.scss (hunk 1 above)
Comment (daviddarnes) [reply assets/styles.scss]: @odow strange you couldn't change it, guessing you were using the correct build commands? I'll change it to a tint of orange.

File: _layouts/page-with-notification.html (hunk 2 above)
Comment (daviddarnes) [reply _layouts/page-with-notification.html]: @mlubin sure! Will shift it over

File: _config.yml (hunk 3 above)
Comment (daviddarnes) [reply _config.yml]: @mlubin @odow I've changed this now so you can just comment out the notification in the config file to turn it off. No need for that default stuff anymore

File: _includes/nav-header.html (hunk 4 above)
Comment (daviddarnes) [reply _includes/nav-header.html]: @mlubin the js does the nav toggle on mobile, hiding the nav on mobile and listening for clicks on the nav button. Jekyll lets you overwrite includes, I needed to do it to add the dropdown. Alembic has this same js in it's `nav-header.html`, so I had to copy it in too 🙂

**Comment by daviddarnes** [comment]
@mlubin I've put in a fix for the flickering nav issue. Also changed the banner colour and moved the message to the config. Now that the notification is in the config I was able to use it as a flag, meaning if there's no notification there's no banner. Also meant that we didn't need the extra layout file and we could overwrite the regular page layout. As an extra bit I've allowed for markdown use in the notification 👍

**Comment by daviddarnes** [comment]
@odow just pushed a fix to sort that out 👍

**Comment by daviddarnes** [comment]
@odow I've just done some tweaking so the notification banner appears on all pages. Is that what you wanted?

**Comment by mlubin** [comment]
@daviddarnes, thanks for the quick fixes. I'd like to try out the new changes but I have a crazy schedule this week. I'll plan to do it over the weekend.

**Comment by mlubin** [comment]
Looks great, thanks!!

**Pull request merged**